<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JapanQuiz</class>
 <widget class="QMainWindow" name="JapanQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JapanQuiz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="startPage">
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dropDownMode">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>91</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Romaji</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kana</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kanji</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBoxLevel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="levelOne">
     <widget class="QPushButton" name="ansButton1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>112</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ansButton2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>112</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ansButton3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>210</y>
        <width>112</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="levelTwo">
     <widget class="QLineEdit" name="lineEditAnswer">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuestion">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxxxxxxxxxxxxxx</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdScore">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTotal">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
